malasakit-django/pcari/views.py

```diff
@@ -234,9 +226,9 @@ def personal_information(request):
 @language_selectable
 def quantitative_questions(request):
     questions = QuantitativeQuestion.objects.all()
-    question_text = [(question.id, question.prompt, question.left_text,
-                      question.right_text) for question in questions]
-    context = {'questions': question_text}
+    question_attrs = [(q.id, q.prompt, q.input_type, q.minval, q.maxval,
+                q.left_text, q.right_text) for q in questions]
+    context = {'questions': question_attrs}
```

Comment:
Rather than stacking the attributes of the question instance into a long tuple, just pass the list of questions directly through the context and use the attributes directly in the template (e.g., `{% for question in questions %}`, then `{{ question.prompt }}`). Avoid one-letter variables.